### PR TITLE
Tango :: 2.2 fixes

### PR DESCRIPTION
**Pull request by ronso0**
some small improvements to make Tango look like it did with qt4.
also, now the crossfader is visible only visible if the (channel) mixer is shown.

### CODE REVIEW

File: res/skins/Tango/skin_settings.xml [hunk 1]
```diff
@@ -647,7 +640,7 @@ Description:
             <ObjectName>SkinSettingsCategory</ObjectName>
             <MinimumSize>200,26</MinimumSize>
             <MaximumSize>200,54</MaximumSize>
-            <SizePolicy>me,min</SizePolicy>
+            <SizePolicy>me,</SizePolicy>
```
Comment (beenisss) [inline res/skins/Tango/skin_settings.xml]: This looks syntactically incorrect but I could be wrong.

**Comment by beenisss** [comment]
Testing on Mac, these look fine to me. I can't actually see any discernible difference between the old and updated skins, not sure if that's a good thing or not.

I don't use any FX or samplers on a regular basis so would be worth double checking with somebody who uses them a lot.

**Comment by beenisss** [comment]
The bpm library column has that same issue I mentioned on Zulip but I assume that is not relevant.

**Comment by ronso0** [comment]
> Testing on Mac, these look fine to me. I can't actually see any discernible difference between the old and updated skins, not sure if that's a good thing or not.

expand the beatgrid buttons (in waveforms) and maximize the library, then you shoud see it.
yes, apart from that that it's just small issues and error messages I fixed.

**Comment by beenisss** [comment]
> expand the beatgrid buttons (in waveforms) and maximize the library, then you shoud see it.

Ah, I get you. Yup, both looking spot on.

**Review by benis — APPROVED** [review]

**Comment by ronso0** [comment]
Ready to merge.

**Comment by uklotzde** [comment]
Still LGTM. I'm planning a new snapshot build for RPM Fusion and would like to include all these nice polishing changes.

**Pull request merged**